lib/resources/dstu2/document_reference.yaml

```diff
@@ -75,7 +72,6 @@ fields:
       info_link: /millennium/dstu2/proprietary-codes/#code-set-72-event-code
 - name: author
   required: 'No'
-  cardinality: 0..1
   type: Reference (Practitioner)
```

Comment:
List of Reference (Practitioner)

Reply:
[updated](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-eefc10b8b4a495f72e678cead5851448L75)